bubble_sort.py

```diff
@@ -28,9 +28,17 @@ def bubble_sort(lista_numeros: list[int]):
 
     >>> bubble_sort([1, 3, 1])
     [1, 1, 3]
-    """
+    """ 
+    n = len(lista_numeros) 
+    for i in range(n): 
+        Troca = False 
+        for j in range(n-i-1): 
+            if lista_numeros[j] > lista_numeros[j+1] : 
+                lista_numeros[j], lista_numeros[j+1] = lista_numeros[j+1], lista_numeros[j]
+                Troca = True
+            if Troca == False: 
+                break
```

Comment:
```suggestion
        if Troca == False: 
            break
```
Aqui vocês só querem comparar FORA do laço do `for j in ...`